security/container_security/security-registries.adoc

```diff
@@ -0,0 +1,47 @@
+[id="security-registries"]
+= Registries
+include::modules/common-attributes.adoc[]
+:context: security-registries
+
+toc::[]
+
+Container registries store container images to:
+
+* Make images accessible to others
+* Organize images into repositories that can include multiple versions of an image
+* Optionally limit access to images (based on different authentication methods) or
+make them publicly available
+
+There are public container registries, such as the Quay.io and Docker Hub
```

Comment:
Should "the" in front of "Quay.io" be removed?

Reply:
Oh, yes. That happened when I moved the sentence around. Done.